source/gui/_addonStoreGui/viewModels/store.py

```diff
@@ -241,6 +241,21 @@ def _makeActionsList(self):
 				validCheck=lambda aVM: isinstance(aVM.model, _AddonStoreModel),
 				actionTarget=selectedListItem
 			),
+			AddonActionVM(
+				# Translators: Label for an action that opens the webpage to see and send feedback for the selected add-on
+				displayName=pgettext("addonStore", "Re&views"),

```

Comment:
Maybe this could be "Community reviews" to clear the misconception surrounding developer reviews

Reply:
"User reviews" might be a bit more standard.

I do prefer some usage of "reviews" over "feedback", however--Feedback suggests an interaction between the users/community and the author. Reviews suggest comments meant to be read by other users, at least in the way it's utilized by Amazon, Ebay, Google, etc.

Reply:
I prefer Community reviews, so that everybody (users, but also developers and others) are encouraged to participate.